mux.go

```diff
@@ -253,3 +252,24 @@ func camelToHuman(s string) string {
 	}
 	return result.String()
 }
+
+// DefaultDescription returns a default .md description for a controller
+func DefaultDescription[T any](handler string, middlewares []T) string {
+	description := "#### Controller: \n\n`" +
+		handler + "`"
+
```

Comment:
This would be more readable I think

```suggestion
	description := fmt.Sprintf ("#### Controller: \n\n`%s`", handler)

```